card/mod/notifications/set/type/notification_template.rb

```diff
@@ -29,5 +29,5 @@ def disappear?
 end
 
 def alert_class
-  contextual_class.present? ? contextual_class_card.item : :success
+  contextual_class.present? ? contextual_class_card.item_names.first : :success
```

Comment:
why did you remove `item`?

Reply:
It's inconsistent API.  we have #item_names, #item_id, and #item_cards.  To be consistent, #item would need to be If we want to support #item_name, #item_id, and #item_card, that would at least be consistent. But given that there's only one usage, it didn't seem worth it.

Reply:
Often we only care about one item. We have a lot of `item_names.first` and `item_cards.first` calls in wikirate and whenever I write it I want something like `item`. I didn't it know it already existed.

Reply:
If it's #item_name, I guess I'm fine with it. #item breaks the pattern.

Reply:
actually, what if it were like:  `def item_name index=0, args`